packages/models/src/lib/implementation/constants.ts

```diff
@@ -1,5 +1,12 @@
-import type { Format } from '../persist-config.js';
+import type { Format, PersistConfig } from '../persist-config.js';
 
 export const DEFAULT_PERSIST_OUTPUT_DIR = '.code-pushup';
 export const DEFAULT_PERSIST_FILENAME = 'report';
 export const DEFAULT_PERSIST_FORMAT: Format[] = ['json', 'md'];
+
+export const DEFAULT_PERSIST_CONFIG: Required<PersistConfig> = {
+  outputDir: DEFAULT_PERSIST_OUTPUT_DIR,
+  filename: DEFAULT_PERSIST_FILENAME,
+  format: DEFAULT_PERSIST_FORMAT,
+  skipReports: false,
+};
```

Comment:
```suggestion
export const DEFAULT_PERSIST_SKIP_REPORT = false;

export const DEFAULT_PERSIST_CONFIG: Required<PersistConfig> = {
  outputDir: DEFAULT_PERSIST_OUTPUT_DIR,
  filename: DEFAULT_PERSIST_FILENAME,
  format: DEFAULT_PERSIST_FORMAT,
  skipReports: DEFAULT_PERSIST_SKIP_REPORT,
};
```

This way we can reuse it in other areas like we do with `DEFAULT_PERSIST_OUTPUT_DIR`